VMD_scripts/Blob_GUI.tcl

```diff
@@ -73,13 +129,23 @@ proc ::blobulator::GUI {} {
 #	::blobulator::blobColorType2 (String): A parameter used for ::blobulator::graphRepOptions checks, if it is set to this variable, will call ::blobulator::graphRepUser2 proc
 #	blobs (Object): Overarching window frame
 proc blobulation {} {
+	if {$::blobulator::checkForSelect == 1} {
```

Comment:
would you ever not check for a selection?

Reply:
Yes, this variable is checking if the checkbox for the selector is active or not.